fullstack/ruby/price_calculator/lib/cart/models/cart_item.rb

```diff
@@ -0,0 +1,16 @@
+class CartItem < Struct.new(:product_type, :options, :artist_markup, :quantity, :base_price)
+  def calculate_price
+    # raise NotImplementedError
+  end
+
+  def self.from_hash(hash)
+    ProductBasePrice.new(
```

Comment:
Is this intentional?

Reply:
I should say yes I intended to do that but it was a copy paste mistake.